kudu/src/main/java/com/yahoo/ycsb/db/KuduYCSBClient.java

```diff
@@ -23,9 +23,13 @@
 import com.yahoo.ycsb.Status;
 import com.yahoo.ycsb.StringByteIterator;
 import com.yahoo.ycsb.workloads.CoreWorkload;
-import org.kududb.ColumnSchema;
-import org.kududb.Schema;
-import org.kududb.client.*;
+import org.slf4j.Logger;
```

Comment:
Should be above the license with a line like:

```
Copyright (c) 2016 YCSB contributors. All rights reserved.
```

See `kudu/README.md` for an example.